### ISSUE TITLE
Dist SCSS file does not contain variables

### ISSUE THREAD

**Issue by alexrussell** [ktsn/vue-range-slider#28]
I am following the readme file for customising the styles of the component ([example](https://github.com/ktsn/vue-range-slider/blob/master/example/main.scss)), but none of my variables are overriding the defaults. Looking into it, it's because the dist `scss` file that is generated on publish/install is already 'transpiled' to CSS and is not actually a SCSS file. It doesn't have the autoprefixer treatment that the CSS file does so hasn't fully undergone the process, but all the variables are gone and the variables placeholders in the file are filled with actual values.

Has something changed in the rollup configuration since you implemented the SCSS variables?

See below for reproduction:

```sh
❯ mkdir test
❯ cd test
❯ npm install vue-range-slider

+ vue-range-slider@0.5.0
added 1 package in 1.112s

❯ cat node_modules/vue-range-slider/dist/vue-range-slider.scss
.range-slider {
  display: inline-block;
  padding: 0 10px;
  height: 20px;
  width: 130px; }
...
```

**Comment by ktsn** [comment]
Fixed in v0.5.1

**Comment by alexrussell** [comment]
Wahey, fast. Thanks, I'll test it out tomorrow!
